pybossa/api/__init__.py

```diff
@@ -290,7 +301,7 @@ def _retrieve_new_task(project_id, task_id=None):
     # Allow scheduling a gold-only task if quiz mode is enabled for the user and the project.
     quiz_mode_enabled = user.get_quiz_in_progress(project) and project.info["quiz"]["enabled"]
 
-    task = sched.new_task(project.id,
+    tasks = sched.new_task(project.id,
```

Comment:
Why is this `task` renamed to 'tasks`? Looks like it's still a single task

Reply:
It could be multiple tasks, depending on the value of ```limit```. See code ```[new_task]```https://github.com/bloomberg/pybossa/blob/13ccfc8a4d9708c135738721f394fcb2684e9083/pybossa/api/__init__.py#L293
Then ```get_locked_task``` -> ```locked_task_sql```
https://github.com/bloomberg/pybossa/blob/13ccfc8a4d9708c135738721f394fcb2684e9083/pybossa/sched.py#L496-L514

Reply:
I see, thanks

Reply:
I realized a few lines after the code mentioned in the previous comment, the code has early return which will result a list with a single task:
https://github.com/bloomberg/pybossa/blob/13ccfc8a4d9708c135738721f394fcb2684e9083/pybossa/sched.py#L362
I will change back to the singular name. Thanks for pointing this out.